docs/appkit/vue/solana/about/implementation.mdx

```diff
@@ -5,8 +5,8 @@ In your `App.vue` file set up the following configuration
 ```html
 <script setup>
   
-  import { createAppKit, defaultSolanaConfig } from '@web3modal/solana/vue'
-  import { solana, solanaTestnet, solanaDevnet } from '@web3modal/solana/chains'
+  import { createAppKit, defaultSolanaConfig } from '@reown/solana/vue'
+  import { solana, solanaTestnet, solanaDevnet } from '@reown/solana/chains'
```

Comment:
```suggestion
  import { createAppKit } from '@reown/appkit/vue'
  import { solana, solanaTestnet, solanaDevnet } from '@reown/appkit/networks'
```

`defaultSolanaConfig` is no longer required with AppKit. The Solana networks can be imported from `@reown/appkit/networks`